ipaserver/plugins/domainlevel.py

```diff
@@ -131,6 +155,13 @@ def execute(self, *args, **options):
                             .format(desired_value, master['cn'][0]))
                 raise errors.InvalidDomainLevelError(reason=message)
 
+        # Check if conflict entries exist in topology subtree, should be resolved first
+        if check_conflict_entries(ldap, self.api):
```

Comment:
You can then just plainly call `check_conflict_entries(ldap, desired_value, self.api)` here.